src/client/components/Appheader/AppHeader.component.js

```diff
@@ -0,0 +1,20 @@
+// dumb/statless img component with props
+
+import React from 'react';
+import './AppHeader.style.css';
+import PropTypes from 'prop-types';
+
+export const AppHeader = (props) => {
+  return (
+    <div className="AppHeader-div">
+      <img src={props.path} alt="AppHeader" />
```

Comment:
usually, it would be a bit easier to name the prop the same as what the `<img>` actually consumes.

so I would have named it `props.src` (but nothing wrong with this approach)